<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>crearProfesor</class>
 <widget class="QWidget" name="crearProfesor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1106</width>
    <height>907</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>crearProfesor</string>
  </property>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>50</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Crear Profesor</string>
   </property>
  </widget>
  <widget class="QPushButton" name="botonCrear">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>440</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>CREAR</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nomProfesor">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>170</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="labelNom">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>120</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Nombre del Profesor:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelDoc">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>120</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Doctor?</string>
   </property>
  </widget>
  <widget class="QLabel" name="LabelNote">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>150</y>
     <width>251</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>(Marcar en caso de que tenga un Doctorado)</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="trueDoc">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>190</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Doctor</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelgrado">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>260</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Grado al que pertenece:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelAlumnos">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>220</y>
     <width>311</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>NÃºmero de Alumnos que acepta</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="numAlumnos">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>310</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="maximum">
    <number>15</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="nomGrado">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>310</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>botonCrear</sender>
   <signal>clicked()</signal>
   <receiver>crearProfesor</receiver>
   <slot>crearClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>439</x>
     <y>467</y>
    </hint>
    <hint type="destinationlabel">
     <x>431</x>
     <y>527</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>crearClicked()</slot>
  <slot>backClicked()</slot>
 </slots>
</ui>
